panel/src/components/Views/Languages/LanguageView.vue

```diff
@@ -91,15 +85,6 @@ export default {
 				)}/${option}`
 			);
 		},
-		update(focus) {
-			this.$dialog(`languages/${this.id}/update`, {
-				on: {
-					ready: () => {
-						this.$panel.dialog.focus(focus);
-					}
-				}
-			});
-		},
```

Comment:
We can remove this and just use `$dialog(`languages/${this.id}/update`)` as the `focus` parameter was never passed.